packages/core/index.ts

```diff
@@ -91,8 +91,8 @@ export interface StrictOptions<
 
 export function createEnv<
   TPrefix extends string,
-  TServer extends Record<string, ZodType>,
-  TClient extends Record<string, ZodType>
+  TServer extends Record<string, ZodType> = NonNullable<unknown>,
```

Comment:
This is my first time using `NonNullable<unknown>`, an alternative to `{}`, as the ESLint rule doesn't allow me to use it🤷

Reply:
`Record<string, never>` is the type for `{}` that eslint allows fyi

Reply:
Yeah, I was using that originally, but unfortunately it didn't work in this case.